draft-ietf-quic-recovery.md

```diff
@@ -994,11 +994,11 @@ congestion without depending on PTO expiration.
 
 ### Establishing Persistent Congestion
 
-A sender establishes persistent congestion after the receipt of an
-acknowledgement if at least two ack-eliciting packets are declared lost, and:
+A sender MUST establish persistent congestion after the receipt of an
```

Comment:
MUST isn't necessary here.

```suggestion
A sender establishes persistent congestion after the receipt of an
```

Reply:
@martinthomson : Can you clarify why you think so? If you are thinking about enforceability, this document has several other unenforceable MUSTs, especially in the congestion controller.